pennylane/grouping/group_observables.py

```diff
@@ -80,8 +80,8 @@ def __init__(self, observables, grouping_type="qwc", graph_colourer="rlf"):
 
         self.graph_colourer = GRAPH_COLOURING_METHODS[graph_colourer.lower()]
         self.observables = observables
-        self.__wire_map = None
-        self.__n_qubits = None
+        self._wire_map = None
+        self._n_qubits = None
```

Comment:
@ral9000 I changed these since the double underline was interfering with a test I added to increase coverage. Are they actually needed? Or are they there in anticipating of being needed for a future addition?

Reply:
These were included to offer flexibility. They are completely optional, and don't need to be specified if `PauliGroupingStrategy` is only being used in the measurement optimization techniques.